Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Modifications Copyright 2024 SAP SE or an SAP affiliate company and Gardener contributors
+
+#############      builder       #############
+FROM golang:1.21.7 AS builder
+ARG GOPROXY=https://proxy.golang.org,direct
+ENV GOPROXY=$GOPROXY
+WORKDIR /go/src/github.com/plkokanov/quic-reverse-http-tunnel
+COPY . .
+RUN make install
+
+#############      server     #############
+FROM alpine:3.18.4 AS server
+RUN apk add --update tzdata
```

Comment:
I tried `gcr.io/distroless/static-debian12:nonroot` and have not seen any issues.

Reply:
Thanks, I was wondering whether to use distroless, but ended not trying it out because of the `tzdata` addition. I will replace the image - `tzdata` should not be needed with distroless right?

Reply:
`tzdata` is installed in distroless, ref: https://github.com/GoogleContainerTools/distroless/blob/main/base/README.md

Reply:
`tzdata` is part of distroless, see [here](https://github.com/GoogleContainerTools/distroless/blob/main/base/README.md).